src/wp-admin/css/about.css

```diff
@@ -684,12 +689,11 @@
 	.about__header {
 		min-height: auto;
 	}
-}
 
-@media screen and (max-width: 480px) {
```

Comment:
Is this change part of the PR or was it changed by mistake? If you made the change, there might be some CSS issues on small screens.

Reply:
This is intentional, because there will be one more item in the menu, it is getting bigger and styles which were applied to max-width 480px now need to be applied early for max-width 600px. I will make screenshots add att them to the ticket.

Reply:
That would be great.